### ISSUE TITLE
questions about the ligtning branch

### ISSUE THREAD

**Issue by tyrink** [ToTheBeginning/PuLID#28]
what's the relationship between the Conventional Diffusion branch and the Lightning T2I branch? Do they share the same UNet weights but with different sampling algorithms?

**Comment by ToTheBeginning** [comment]
Yes, they share the same weights but with different sampling algorithms.

**Comment by tyrink** [comment]
How is the Cid condition fed into the Lightning T2I branch? Lcm-Lightning model should not include the adapter structure for id injection? And for contrast alignment loss calculation, does the average loss value of the four denoising steps of lcm need to be calculated for the lightning branch in each training iter?

**Comment by FORRYKIM** [comment]
have you solved these problems?

**Comment by tyrink** [comment]
> have you solved these problems?

Not yet
